zanata-war/src/main/java/org/zanata/webtrans/server/HibernateIntegrator.java

```diff
@@ -37,6 +38,8 @@ public void integrate(Configuration configuration,
                     updateListener);
             eventListenerRegistry.appendListeners(EventType.POST_INSERT,
                     updateListener);
+            eventListenerRegistry.appendListeners(EventType.POST_COMMIT_UPDATE,
+                    ServiceLocator.instance().getInstance(SlugEntityUpdatedListener.class));
```

Comment:
What's the benefit this will give us versus making it an application scoped bean? The POJO will be a singleton as well (hibernate will only use one instance of a listener for all its events AFAIK)